src/pages/Login.tsx

```diff
@@ -22,9 +22,14 @@ const Login = () => (
             <Form.Control type="password" />
           </Form.Group>
           <Form.Text className="text-muted">
-            <p>Not a member yet? <a className="orangeText" href="/sign-up">Join Us</a></p>
+            <p>
+              Not a member yet?{' '}
+              <a className="orangeText" href="/sign-up">
```

Comment:
I noticed you replaced the hrefs in App.tsx with "Link" and "to". Shouldn't we do that here too then?

Reply:
@CodyNicholson yeah, I'll do more here once we actually touch the login/registration flow. The Links I changed now were affecting the header navigation sometimes and causing flickering, so that's why I changed it there for now